### DISCUSSION TITLE
Can't change World

### DISCUSSION THREAD

**Discussion by T0mk4** [lloesche/valheim-server-docker#333] [Q&A]
Hi. I created a new seed locally, copied it to my Qnap and changed the WORLD_NAME to the new world, but the Server always loads the old world as i can read from the server log and see it ingame. I deleted the old world files and just left the new ones in the folder. The server is recreating the old world everytime. What am i doing wrong?

**Comment by lloesche** [comment]
Did you rename the world by any chance? Valheim stores the original name in the .fwl file and does not load the world if the file has been renamed between when you originally generated it and when it's copied onto the server.

**Reply by T0mk4** [reply]
yep. I did everything 100% correct. I deleted the container now and let Qnap recreate it with the exact yaml and env file and now it works as it should.
